jenkins/tagging/Jenkinsfile

```diff
@@ -1,4 +1,7 @@
 pipeline {
+    options {
+        disableConcurrentBuilds()
+    }
```

Comment:
Did you test it, if it work lets enable it in all the jobs

Reply:
yes, worked.